### ISSUE TITLE
Graph Transform Tool

### ISSUE THREAD

**Issue by Deep-Blue-2013** [SciSharp/TensorFlow.NET#475]
When you have finished training a model and want to deploy it in production, you'll often want to modify it to better run in its final environment. For example if you're targeting a phone you might want to shrink the file size by quantizing the weights, or optimize away batch normalization or other training-only features. The Graph Transform framework offers a suite of tools for modifying computational graphs, and a framework to make it easy to write your own modifications.

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/graph_transforms/README.md
